src/lsqecc/pauli_rotations/circuit.py

```diff
@@ -191,15 +191,62 @@ def remove_y_operators_from_circuit(self, start_index: int = 0) -> None:
 
             i += 1
 
-    def commute_pi_over_four_rotation(self, index: int) -> None:
-        """Commute a rotation block pass its neighbor block.
+    def __swap_adjacent_commuting_blocks(self, index: int) -> None:
```

Comment:
I think were using the convention to use a single underscore for private methods, but it doesn't really matter